src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/AuthDAO.scala

```diff
@@ -18,6 +18,8 @@ trait AuthDAO {
 
   def loadGroupByEmail(groupEmail: String, txn: RawlsTransaction): Option[RawlsGroup]
 
+  def findRelatedWorkspaces(groupRef: RawlsGroupRef, txn: RawlsTransaction): Seq[Workspace]
```

Comment:
findWorkspacesForGroup is good. I'll also add a comment. It finds all of the workspaces that have the specified group in any part of its ACL tree.